### PR TITLE
Add an empty unit for CSD 7 - the new ai ml unit

### PR DESCRIPTION
**Pull request by dmcavoy**
This will add an empty unit for CSD Unit 7.  It is clickable. You can go to the page. You can try to view it in the progress view. Below are screenshots of what it will look like.  

Screen Shot 2021-05-17 at 4 51 28 PM
Screen Shot 2021-05-17 at 4 44 55 PM
Screen Shot 2021-05-17 at 4 44 49 PM
Screen Shot 2021-05-17 at 4 44 44 PM

### CODE REVIEW

File: dashboard/config/courses/csd-2021.course [hunk 1]
```diff
@@ -82,95 +50,40 @@
...
+        "is_rollup": true
       },
       "seeding_key": {
-        "resource.key": "csd_guide_to_debugging"
+        "resource.key": "all_vocabulary_1"
       }
     }
   ],
   "student_resources": [
-    {
```
Comment (bethanyaconnor) [inline dashboard/config/courses/csd-2021.course]: Did you mean to remove all the student resources?

File: lib/cdo/script_constants.rb [hunk 2]
```diff
@@ -136,6 +136,7 @@ module ScriptConstants
       CSD4_2021_NAME = 'csd4-2021'.freeze,
       CSD5_2021_NAME = 'csd5-2021'.freeze,
       CSD6_2021_NAME = 'csd6-2021'.freeze,
+      CSD6_2021_NAME = 'csd7-2021'.freeze,
```
Comment (bethanyaconnor) [inline lib/cdo/script_constants.rb]: Should be `CSD7_2021_NAME` right?

Comment (dmcavoy) [reply lib/cdo/script_constants.rb]: Thank you for having a brain that is actually working today and catching all my mistakes :)

File: dashboard/config/courses/csd-2021.course (hunk 1 above)
Comment (dmcavoy) [reply dashboard/config/courses/csd-2021.course]: No... I see whats going on

**Review by bethanyaconnor — APPROVED** [review]
Change looks good! I seeded it on my dev environment and everything seems to work okay.

My one concern is the pdfs. I don't know if there will be any issues if there are no lessons or what the resulting pdf will look like

**Comment by dmcavoy** [comment]
> Change looks good! I seeded it on my dev environment and everything seems to work okay.
> 
> My one concern is the pdfs. I don't know if there will be any issues if there are no lessons or what the resulting pdf will look like

@Hamms any thoughts here?

**Comment by tess323** [comment]
This looks good to me - also reached out to Dan to run it by him

**Comment by dmcavoy** [comment]
I'm going to put it in staging now so it makes it to levelbuilder with tomorrows DTP and DTL.

**Comment by dancodedotorg** [comment]
👋 Hi folks! Dani - this looks great! I wonder if we can update the text beyond 'coming soon'? Maybe to this instead:

> This unit is a hands-on introduction to developing a machine learning model with tabular data. You will explore how computers learn from data to make decisions, then develop your own machine learning projects around real-world data. The unit culminates with designing your own machine learning app to solve a personally relevant problem.

>Implementation guidance for Unit 7
- Complete Unit 1 Chapter 1 & Unit 4 before starting this unit
- This is an optional unit that can be used in place of another CSD unit
- View the full [CS Discoveries implementation guide](https://docs.google.com/document/d/1u-eIlB2cAwMuSckzbMWkteQIcsXgez7FTrbjP8PBB4o/preview)

> **Coming Soon!** This unit will be available in July 2021!

**Comment by dancodedotorg** [comment]
Whoops - I'm too late =P

**Comment by dmcavoy** [comment]
@dancodedotorg You can update it when it hits levelbuilder :)

**Comment by dancodedotorg** [comment]
Oh! Ha! Perfect! Thank youuuu!

**Pull request merged**